recipes/sdl2_mixer/CmakeLists.txt

```diff
@@ -0,0 +1,236 @@
+# Written by Daniel Schreij (dschreij@gmail.com) on 1 Nov 2016
+# This procedure is configured to work in an Anaconda build environment (contiuum.io)
```

Comment:
They are on mercurial, and I currently don't have the time to figure out how that system works, or how to submit a PR. Is there another easy way to do this which you can suggest? Their support forum is quite nailed down as well surprisingly (as in, even though I do have an account, I'm not allowed to post?).